src/util.h

```diff
@@ -30,6 +30,7 @@
 #ifndef __REDIS_UTIL_H
 #define __REDIS_UTIL_H
 
+#include <fcntl.h>
```

Comment:
This include isn't needed in util.h now.

Reply:
This is for macOS build, `O_CLOEXEC` and `O_NONBLOCK` will be not defined without this include.

Reply:
These macros are not used in this file. The include is needed in util.c though.

Reply:
I was talking about https://github.com/redis/redis/runs/3629070443?check_suite_focus=true